### ISSUE TITLE
[5.x]: Manual sorting of routes is not accessible

### ISSUE THREAD

**Issue by MoritzLost** [craftcms/cms#17687]
### Description

I was looking for a way to define the order of precedences for routes defined in Settings → Routes in the Control Panel. I discovered by accident that you can drag-and-drop the routes to rearrange them, and that the order in which they are placed determines the precedence if more than one route can apply to a given URL.

I can't find any documentation for this feature, and the interface doesn't provide any affordance for this UI interaction.

### Expected behavior

- Add drag handles to the routes to indicate that they can be rearranged through drag-and-drop, similar to elements in a structure.
- The fact that the order of the routes determines the order of precedence should be noted on that page itself, and/or in the [documentation on dynamic routes](https://craftcms.com/docs/5.x/system/routing.html#dynamic-routes).
- I don't think there's a keyboard alternative for the drag-and-drop interaction, that should probably be added as well?

### Craft CMS version

5.8.11

### PHP version

8.3

### Installed plugins and versions

-

**Comment by i-just** [comment]
Hi Moritz, thanks for reporting! I raised a PR to add the drag handle and actions menu with the reorder options.

**Comment by brandonkelly** [comment]
Craft 5.9.0 is out with that fix 🎉
